app/cafe.py

```diff
@@ -0,0 +1,17 @@
+import datetime
```

Comment:
fix imports ordering
![Screenshot from 2023-10-13 13-38-45](https://github.com/mate-academy/py-lockdown-cafe/assets/122517203/df8a51be-25f2-47cb-a92c-cae2f6f81408)

Reply:
not fixed, You should put a blank line between each group of imports